README.md

```diff
@@ -1,5 +1,7 @@
 # knife-supermarket
 
+This Plugin is DEPRECATED: the code has been moved into core Chef.
```

Comment:
Yeah, I wasn't convinced anyone would see the README so I didn't write that many words. Probably better to.